src/utils/product.ts

```diff
@@ -8,8 +8,8 @@ import { ESProductType, JsonApiDocument, JsonApiResponse, JsonApiSingleResponse
 const productCustomAttributesPrefix = 'prodattr_'
 const productOptionAttributePrefix = 'prodopt_'
 
-const generateOptionAttributeCode = (attributeIdentifier) => `${productOptionAttributePrefix}${attributeIdentifier}`
-const generateCustomAttributeCode = (attributeIdentifier) => `${productCustomAttributesPrefix}${attributeIdentifier}`
+const generateOptionAttributeCode = (attributeIdentifier) => `${attributeIdentifier}`
```

Comment:
@arudnicka Is there a reason for this function to exist? We're taking `attributeIdentifier` and return it. It's not doing anything.

Reply:
Tom wrote them for safety, but I think they are unnecessary. These functions are used many times and I don't want to remove them for now. I will remove them in separate PR, if everything will work properly after testing without them.